_beats/testing/environments/latest.yml

```diff
@@ -26,7 +26,7 @@ services:
       - ./docker/logstash/pki:/etc/pki:ro
 
   kibana:
-    image: docker.elastic.co/kibana/kibana:7.2.0
+    image: docker.elastic.co/kibana/kibana:7.3.2
```

Comment:
Shouldn't that be `7.4.0` now (I know it is inherited, but might be worth a manual change until updated).

Reply:
:( good catch